src/osdc/Objecter.cc

```diff
@@ -1544,7 +1544,7 @@ void Objecter::_check_op_pool_dne(Op *op, unique_lock *sl)
 
       OSDSession *s = op->session;
       if (s) {
-	assert(s != NULL);
+	assert(s);
```

Comment:
What is the use of an assert here, as `s` always evaluates to `true` ?

Reply:
@shinobu-x could you take this chance to remove this `assert()`?